ext/node/global.rs

```diff
@@ -80,7 +81,7 @@ const MANAGED_GLOBALS: [&[u16]; 13] = [
 ];
 
 const SHORTEST_MANAGED_GLOBAL: usize = 4;
-const LONGEST_MANAGED_GLOBAL: usize = 14;
+const LONGEST_MANAGED_GLOBAL: usize = 17;
```

Comment:
these could be calculated in a const expr so they can't fall out of date

Reply:
Is that easily/reasonably done? Looked into `const fn` feature, but it seems very limited..

Anyway this value is indirectly tested via the test cases